llarp/peerstats/peer_db.cpp

```diff
@@ -0,0 +1,269 @@
+#include <peerstats/peer_db.hpp>
+
+#include <util/logging/logger.hpp>
+#include <util/status.hpp>
+#include <util/str.hpp>
+
+namespace llarp
+{
+  PeerDb::PeerDb()
+  {
+    m_lastFlush.store({});
+  }
+
+  void
+  PeerDb::loadDatabase(std::optional<std::filesystem::path> file)
```

Comment:
use `fs::path` because macos is broke af

Reply:
fixed (d2a27c9b)